community_server/src/Model/Table/TransactionsTable.php

```diff
@@ -206,7 +210,7 @@ public function listTransactionsHumanReadable($stateUserTransactions, array $use
                           'decay_start' => $calculated_decay['start_date'],
                           'decay_end' => $calculated_decay['end_date']
                       ]; 
-                      if($prev->transaction_id < $decay_start_transaction_id && 
+                      if($decay_start_time && $prev->transaction_id < $decay_start_transaction_id && 
```

Comment:
`$decay_start_time > 0` ? Or is `0 == false` in any case?

Reply:
$decay_start_time is initalized with 0 and set at a actual value, if an decay block exist